toolchain/check/cpp/import.cpp

```diff
@@ -1727,16 +1732,20 @@ static auto AddDependentUnimportedTypeDecls(const Context& context,
   }
 
   if (const auto* tag_type = type->getAs<clang::TagType>()) {
-    AddDependentDecl(context, tag_type->getOriginalDecl(), worklist);
+    AddDependentDecl(context, SemIR::ClangDeclKey(tag_type->getOriginalDecl()),
+                     worklist);
   }
 }
 
 // Finds all decls that need to be imported before importing the given function
 // and adds them to the given set.
 static auto AddDependentUnimportedFunctionDecls(
-    const Context& context, const clang::FunctionDecl& clang_decl,
+    const Context& context, const clang::FunctionDecl& clang_decl, int params,
```

Comment:
Perhaps pass as `num_params` to distinguish more from `parameters`?

```suggestion
    const Context& context, const clang::FunctionDecl& clang_decl, int num_params,
```